MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -590,6 +591,20 @@ namespace WPEFramework {
                 maintenanceManagerOnBootup();
             }
         }
+	void MaintenanceManager::sendIARMmode(const char *eventname,int *mode, int size)
```

Comment:
You can directly call IARM API inside setMaintenanceMode if this is not being used else where.

Reply:
It may use in future. That's why i have added.